content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -52,7 +52,7 @@ In version 1.9, the CCM runs the following controllers from the preceding list:
 Volume controller was deliberately chosen to not be a part of CCM. Due to the complexity involved and due to the existing efforts to abstract away vendor specific volume logic, it was decided that volume controller will not be moved to CCM.
 {{< /note >}}
 
-The original plan to support volumes using CCM was to use Flex volumes to support pluggable volumes. However, a competing effort known as CSI is being planned to replace Flex.
+The original plan to support volumes using CCM was to use [Flex](https://kubernetes.io/docs/concepts/storage/volumes/#flexVolume) volumes to support pluggable volumes. However, a competing effort known as [CSI](https://kubernetes.io/docs/concepts/storage/volumes/#csi) is being planned to replace Flex.
```

Comment:
nit:
```suggestion
The original plan to support volumes using CCM was to use [Flex](https://kubernetes.io/docs/concepts/storage/volumes/#flexVolume) volumes to support pluggable volumes. However, a competing effort known as [CSI](/docs/concepts/storage/volumes/#csi) is being planned to replace Flex.
```

alternatively: reword this section more comprehensively in order to [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future).

Reply:
In the volume's background https://kubernetes.io/docs/concepts/storage/volumes/#background section it has been described the challenges with volume with respect to docker and kubernetes approach towards volumes, should I include that also in reference.

Reply:
@tanalam2411: what is the issue you're most keen to solve here?

Reply:
Just wanted to add reference of Flex volume and CSI so that reader can read more about it, in case wants to know what flex volumes are and why CSI is taking over it.

Reply:
If you accept the suggestion above, this change is OK to merge.

Reply:
Sure, can you please edit your suggestion as you have updated only one link of CSI, please edit the flex(by removing `https://kubernetes.io` from link) link also and I will commit the suggestion itself. Thanks

Reply:
Ah yes. Updated.

Reply:
Thanks